src/dlgprefreplaygaindlg.ui

```diff
@@ -248,7 +254,7 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>If an unanalyzed track is playing, Mixxx won't apply a newly calculated ReplayGain to avoid an abrupt volume change. </string>
+         <string>If an unanalyzed track is playing, to avoid an abrupt volume change Mixxx won't apply a newly calculated ReplayGain value.</string>
```

Comment:
please restore original, the new sentence is awkward.  or: "When an unanalyzed track is playing, Mixxx will avoid an abrupt volume change by not applying a newly calculated ReplayGain value.